client/app/src/main/java/com/example/criengine/MyBooksActivity.java

```diff
@@ -0,0 +1,137 @@
+        Book book4 = new Book("13", "Ralph", "A book of Lyve", "Author D.", "This is a description.", "1I2SS05", "Available");
+        Book book5 = new Book("14", "Ralph", "A book of Evil", "Author E.", "This is yet another description.", "1I2SS06", "Available");
+        book1.addRequesters("Joe Smith");
+        book2.addBorrower("Bill John");
+        book3.setPotentialBorrower("Cyn Lord");
+        book4.setPotentialBorrower("Cyn Lord");
+        book4.getPotentialBorrower().setHandoffComplete();
+        Book[] books = {book1, book2, book3, book4, book5};
+        myBooks = new ArrayList<Book>();
+        myBooks.addAll(Arrays.asList(books));
+        // DUMMY CODE ENDS HERE.
+
+        // Contains the books that will be displayed on the screen.
+        displayBooks = new ArrayList<Book>();
+
+        // Set the adapter.
+        myBooksListAdapter = new MyBooksAdapter(this, myBooks);
+
+        bookNameTextView = findViewById(R.id.bookListView);
+        bookNameTextView.setAdapter(myBooksListAdapter);
+
+        // Opens to the book information screen when you click on a specific book.
+        bookNameTextView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
+            @Override
+            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+                // FIXME: redirect to book view
+            }
+        });
+
+        // Opens to the add-a-book screen when you click the button.
+        addBookButton = findViewById(R.id.add_a_book);
+        addBookButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                // FIXME: redirect to add book activity.
+            }
+        });
+
+        // Opens the filter fragment where you can filter information.
+        filterButton = findViewById(R.id.filter_button);
+        filterButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                new FilterFragmentActivity(filterStatus).show(getSupportFragmentManager(), "Filter_Status");
+            }
+        });
+    }
+
+    /**
+     * Handles modifying what is displayed on the screen if the user chooses to filter the info.
+     * @param filterStatus Contains the different status' that the user wants to display.
+     */
+    @Override
+    public void onConfirmPressed(ArrayList<String> filterStatus) {
+        this.filterStatus = filterStatus;
+
+        // Wipe the array so we can start anew.
+        displayBooks.clear();
+
+        // The following statements add books based on their status. If it matches the status's that
+        // the user wants, then it will be displayed.
+        if (filterStatus.contains("Available")) {
+            for (int i = 0; i < myBooks.size(); i++) {
+                if (myBooks.get(i).getRequesters().size() == 0 &&
```

Comment:
I think proper style is to have the && be on the next line before the expression

Reply:
I don't think this is that important of an issue

Reply:
I think we need some consistent code styling and I think that having the operator before on newline is standard.
https://www.oracle.com/technetwork/java/codeconventions-150003.pdf page 6